bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -55,6 +55,9 @@ spec:
       securityContext:
         fsGroup: {{ .Values.master.securityContext.fsGroup }}
         runAsUser: {{ .Values.master.securityContext.runAsUser }}
+        {{- if .Values.master.securityContext.seLinuxOptions }}
+        seLinuxOptions: {{- toYaml .Values.master.securityContext.seLinuxOptions | nindent 10 }}
```

Comment:
USe the common.tplvalues functino instead of `toYaml`

Reply:
done